Formula/llnode.rb

```diff
@@ -0,0 +1,48 @@
+class Llnode < Formula
+  desc "LLDB plugin for live/post-mortem debugging of node.js apps"
+  homepage "https://github.com/indutny/llnode"
+  url "https://github.com/indutny/llnode/archive/v1.0.0.tar.gz"
+  sha256 "fcf90201930e52b49d0fa1b369aac7b5b74c3bbffe34f423d5aa5b899267f295"
+
+  resource "gyp" do
```

Comment:
If this is using `gyp` you might need a `depends_on :python => :build` line as well. AFAIK `gyp` isn't friendly with Python 2.6 or less?